ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -15,7 +16,7 @@
       "location" : "https://github.com/mosip/inji-vci-client-ios-swift.git",
       "state" : {
         "branch" : "develop",
-        "revision" : "2c2a3d2c321e11511f4667e3727587dc0cca61cb"
+        "revision" : "e1a08fbd259a1135d067ad52f2d2ce9370061054"
```

Comment:
Please avoid regeneration of Package.resolved.

Reply:
Used the old file from develop.